claat/proto-renderer/generic_renderer.go

```diff
@@ -0,0 +1,19 @@
+package genrenderer
+
+import (
+	"bytes"
+	"fmt"
+	"reflect"
+	"text/template"
+)
+
+// ExecuteTemplate returns the evaluated template per passed templating
+// namespace, based on the passed Codelab proto element type
+func ExecuteTemplate(el interface{}, t *template.Template) string {
+	var w bytes.Buffer
```

Comment:
bytes.NewBuffer

Reply:
per https://golang.org/pkg/bytes/#NewBuffer, this requires an initializing parameter, which we don't really use or need for this method.

I got this "formatting style" per this conference talk (before: https://youtu.be/WkzGQ-Sm_50?t=439, after: https://youtu.be/WkzGQ-Sm_50?t=639)